electron/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "unified-auctions",
+  "version": "1.0.0",
+  "description": "MakerDAO Unified Auctions Electron App",
+  "main": "main.js",
+  "build": {
+    "appId": "com.makerdao.unified-auctions",
+    "mac": {
+      "category": "public.app-category.finance"
+    },
+    "directories": {
+      "output": "out"
+    },
+    "files": [
+      "main.js",
+      "dist/**/*"
+    ]
+  },
+  "scripts": {
+    "preinstall": "cd ../frontend && npm ci && npm run build && npm run generate",
+    "postinstall": "electron-builder install-app-deps",
+    "prepare": "npx copyfiles -a -u 2 \"../frontend/dist/**/*\" .",
+    "start": "npm ci && electron .",
```

Comment:
Does `npm start` starts a development version of the nuxt or what's the point of that command?